src/compas/datastructures/mesh/operations/weld.py

```diff
@@ -4,10 +4,12 @@
 
 import copy
 
+from compas.datastructures.mesh import Mesh

```

Comment:
@brgcode isn't this a potential issue for circular imports (as discussed a few times in the last few weeks)?